GETDIRENTRIES(2):
"As of Mac OS X 10.6, getdirentries() is deprecated, and it is recommended
 that applications use readdir(3) rather than using getdirentries() directly.
 Due to limitations with the system call, getdirentries() will not work with
 64-bit inodes; in order to use getdirentries(), _DARWIN_NO_64_BIT_INODE must
 be defined."
--- configure.orig	2009-04-18 16:41:11.000000000 +0200
+++ configure	2009-09-22 18:07:10.000000000 +0200
@@ -21169,7 +21169,7 @@
     { echo "$as_me:$LINENO: checking for operating system support" >&5
 echo $ECHO_N "checking for operating system support... $ECHO_C" >&6; }
   case "$target_os" in
-  dragonfly*|freebsd*|netbsd*|openbsd*|darwin*)
+  dragonfly*|freebsd*|netbsd*|openbsd*)
         ac_bm_thunar_vfs_os_impl=bsd
     ;;
 
